types/handsontable/test/global.ts

```diff
@@ -0,0 +1,301 @@
+namespace HandsontableGlobalTest {
+  function test_HandsontableInit() {
+    const elem = document.createElement('div');
+    const hot = new Handsontable(elem, {
```

Comment:
TypeScript does not pick up the declarations unless I wrap them in the module declaration. The reason it works in my other project is that I have noImplicitAny set to false. This is even after I change the interface to var.